cljfmt/src/cljfmt/diff.clj

```diff
@@ -16,6 +16,9 @@
        ^java.io.File (apply io/file)
        .getCanonicalPath))
 
+(defn filename-ext [filename]
```

Comment:
Could you move this function to main.clj (and make it private)? It doesn't fit in diff.clj.